charts/karpenter/README.md

```diff
@@ -63,6 +63,7 @@ cosign verify public.ecr.aws/karpenter/karpenter:1.3.2 \
 | hostNetwork | bool | `false` | Bind the pod to the host network. This is required when using a custom CNI. |
 | imagePullPolicy | string | `"IfNotPresent"` | Image pull policy for Docker images. |
 | imagePullSecrets | list | `[]` | Image pull secrets for Docker images. |
+| initContainers | object | `{}` | add additional initContainers to run before karpenter container starts |
```

Comment:
Was this just a gen miss from a previous PR?

Reply:
Yep